packages/cli/src/lib/bundler/config.ts

```diff
@@ -179,7 +179,7 @@ export async function createConfig(
         child_process: false,
 
         /* new ignores */
-        path: false,
+        path: require.resolve('path-browserify'),
```

Comment:
Is there any other workaround for this?

We want to minimise changes to Backstage itself.

Can it not be added to the overlay [here](https://github.com/manstis/backstage/blob/KOGITO-9338/packages/app/webpack.config.js)?

Reply:
I've spent some time on this and I am still digging around to see if I come up with some other way, but, in the meantime, I guess we can merge this PR to move forward with other things. WDYT?